### PR TITLE
Added tests for Home page

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	for (const [button, url] of buttonMap) {
+		await button.click();
+		const newPage = await driver.DriverContext.waitForEvent('page');
+		expect(newPage.url()).toContain(url);
+		await newPage.close();
+	}
+});
+
+test("Check 'See All News' button from the 'Company insights' block on the 'Home' page @Regression @HomePage @TSWEB-1006", async () => {
+	const companyInsightsblock = driver.getByTestId(HomePage.CompanyInsights);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts:156]: mb better to skip this test till blog will be stable?

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

**Review by anastasiia-liashova — APPROVED** [review]

File: ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts (hunk 1 above)
Comment (KotliarovaK) [reply ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts]: done

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

File: ui_tests_playwright/automation/identifiers/MainSite/MainSiteButtons.ts [hunk 2]
```diff
@@ -11,14 +11,9 @@ export default class MainSiteButtons {
 	static BookAMeeting = 'Button-BookAMeeting';
 	static CheckOutHowWeBuiltIt = 'Button-CheckOutHowWeBuildIt';
 	static TalkToAnExpert = 'Button-TalkToAnExpert';
+	static SeeAllNews = 'Button-SeeAllNews';
```
Comment (ysukhova) [inline ui_tests_playwright/automation/identifiers/MainSite/MainSiteButtons.ts:14]: Please start variable names with a lowercase letter.
https://google.github.io/styleguide/tsguide.html#identifiers

File: ui_tests_playwright/automation/preconditionsData/Links/linkedInReviewLinks.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export enum linkedInReviewLinks {
```
Comment (ysukhova) [inline ui_tests_playwright/automation/preconditionsData/Links/linkedInReviewLinks.ts:1]: Please start enum name with uppercase letter - https://google.github.io/styleguide/tsguide.html#identifiers
}

**Pull request merged**